assembly/src/ast/parsers/mod.rs

```diff
@@ -7,14 +7,15 @@ use super::{
 };
 use core::{fmt::Display, ops::RangeBounds};
 
-pub mod adv_ops;
-pub mod field_ops;
-pub mod io_ops;
-pub mod stack_ops;
-pub mod u32_ops;
-
-pub mod constants;
-pub use constants::calculate_const_value;
+mod adv_ops;
```

Comment:
Why does this PR change being public for all those methods?

Reply:
I just noticed that these modules were public but probably didn't need to be. So, I made them private.